[2023-09-26 01:32:48,472] ERROR log_exception() L1449 Exception on /tubo [POST] - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'add_tubo' did not return a valid response. The function either returned None or ended without a return statement.
[2023-09-26 01:32:48,488] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:32:48] "[35m[1mPOST /tubo HTTP/1.1[0m" 500 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:32:51,509] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:32:51] "OPTIONS /export/URL_to_PDF HTTP/1.1" 200 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:32:53,572] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:32:53] "POST /export/URL_to_PDF HTTP/1.1" 200 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:34:13,091] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.7:5000 (Press CTRL+C to quit) - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:34:20,548] ERROR log_exception() L1449 Exception on /tubos [GET] - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'get_tubos' did not return a valid response. The function either returned None or ended without a return statement.
[2023-09-26 01:34:20,559] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:34:20] "[35m[1mGET /tubos HTTP/1.1[0m" 500 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:34:41,644] ERROR log_exception() L1449 Exception on /tubo [POST] - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'add_tubo' did not return a valid response. The function either returned None or ended without a return statement.
[2023-09-26 01:34:41,651] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:34:41] "[35m[1mPOST /tubo HTTP/1.1[0m" 500 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:38:34,729] ERROR log_exception() L1449 Exception on /tubos [GET] - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'get_tubos' did not return a valid response. The function either returned None or ended without a return statement.
[2023-09-26 01:38:34,740] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:38:34] "[35m[1mGET /tubos HTTP/1.1[0m" 500 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:38:54,399] ERROR log_exception() L1449 Exception on /tubo [POST] - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'add_tubo' did not return a valid response. The function either returned None or ended without a return statement.
[2023-09-26 01:38:54,404] INFO _log() L224  127.0.0.1 - - [26/Sep/2023 01:38:54] "[35m[1mPOST /tubo HTTP/1.1[0m" 500 - - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-26 01:43:17,872] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.7:5000 (Press CTRL+C to quit) - call_trace=E:\SIMDUT\GPD\Engenharia de Software\Arquitetura de Software\_Old\meu_app_api\env\Lib\site-packages\werkzeug\_internal.py L224 
